### PR TITLE
Deprecate the Logs Stream app

### LINKED ISSUE #4315: [Logs] Updated docs for Logs UI deprecation
The log stream component will be hidden by default and there's going to be a Kibana advanced setting to enable it. Our docs need to be updated to explain this.

[Kibana guide](https://www.elastic.co/guide/en/kibana/current/observability.html) also needs to be updated

[Project link](https://github.com/elastic/observability-dev/issues/3242)

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
## Description
Logs Stream is being deactivated by default. This PR addresses that, deemphasizes Logs Stream as a way to view and manage logs, but also includes instructions on activating Logs Stream if users do still want to use it.

### Documentation sets edited in this PR

_Check all that apply._

- [x] Stateful (`docs/en/observability/*`)

### Related issue
Closes #4315 

## Checklist

- [x] Product/Engineering Review
- [x] Writer Review

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [x] The concepts in this PR only apply to one doc set (serverless _or_ stateful)

### CODE REVIEW

**Review by tonyghiani — APPROVED** [review]
LGTM 👌

File: docs/en/observability/tail-logs.asciidoc [hunk 1]
```diff
@@ -14,6 +16,19 @@ along with the power of search.
...
 click *Stop streaming* to view historical logs from a specified time range.
 
+[discrete]
+[[activate-logs-stream]]
+== Activate Logs Stream
+
+Because <<explore-logs, Logs Explorer>> is replacing Logs Stream, Logs Stream and the Logs Stream dashboard panel are disabled by default.
+To activate Logs Stream and the Logs Stream dashboard panel complete the following steps:
+
+. Go to **Management** → **Advanced Settings**
```
Comment (bmorelli25) [inline docs/en/observability/tail-logs.asciidoc:26]:
```suggestion
. Go to **Management** → **Advanced Settings**.
```

File: docs/en/observability/tail-logs.asciidoc [hunk 2]
```diff
@@ -14,6 +16,19 @@ along with the power of search.
...
+. Enter _Logs Stream_ in the search bar.
+. Turn on **Logs Stream**.
+
+After saving your settings, you'll see Logs Stream in the Observability navigation, and the Logs Stream dashboard panel will be available.
+
```
Comment (bmorelli25) [inline docs/en/observability/tail-logs.asciidoc:31]:
```suggestion
```

File: docs/en/observability/tail-logs.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/en/observability/tail-logs.asciidoc:26]: Added this while on a call with Edu to show him how suggestions work. We can ignore.

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**